js/utils/utils.js

```diff
@@ -21,3 +23,12 @@ function createNewTag(newTag) {
   plusses = '<div class="plus">' + "+" + "</div>";
   document.getElementById("tags").innerHTML += tagDiv;
 }
+
+function toggleElementVisibility(elemId, show, prop) {
```

Comment:
If you think you are going to use a lot of times this function to toggle 'block' elements you can set the 3rd parameter as a default/optional one:

```
function toggleElementVisibility(elemId, show, prop = "block") {
  document.getElementById(elemId).style.display = show ? prop : "none";
}
```